### ISSUE TITLE
Value axis scale not optimal for stacked column chart

### ISSUE THREAD

**Issue by Mitwoc** [amcharts/amcharts5#1129]
I have a stacked column chart using a value axis and when I zoom into some specific date range, the scale on the value axis is not optimal. It is calculated according to the global max value, not the local one.

In the attached demo I zoom to the first 10 days of the graph (5 seconds after the graph is loaded), but the y-axis scale is not updated according to the visible data. When I scroll to the left, sometimes it gets better, but most of the time the scale remains according to the global maximum value.
![image](https://github.com/amcharts/amcharts5/assets/10099661/db23c9f8-e240-4c64-ae5a-c9fc03538a57)

Demo: https://codepen.io/Mitwoc/pen/rNoPgjE

How can I change this behavior to scale the y-axis according to visible data?

**Comment by zeroin** [comment]
Indeed, it's a problem on our side. You can solve it if you add 0 values for all the series to data items where you don't have values, but it shouldn't be like this - we will investigate and will let you know.

**Comment by martynasma** [comment]
Fixed in [5.5.0](https://github.com/amcharts/amcharts5/releases/tag/5.5.0).

## [5.5.0] - 2023-10-17

### Added
- `zoomTo` (values: `"start"` and `"end"` (default)) added to `PeriodSelector`. If set to `"start"`, period buttons will selectg from the start of the data, not from the end. `"YTD"` will still select to the end. [More info](https://www.amcharts.com/docs/v5/charts/stock/toolbar/period-selector/#Zoom_anchor_point).
- `PeriodSelector` now supports `"custom"` periods. [More info](https://www.amcharts.com/docs/v5/charts/stock/toolbar/period-selector/#Custom_periods).
- New filter settings added to `Sprite`: `blur`, `brightness`, `contrast`, `saturate`, `sepia`, `invert`, `hue`. [More info](https://www.amcharts.com/docs/v5/concepts/colors-gradients-and-patterns/filters/#Built_in_filters). **These filters are not supported by Safari browsers.**
- New `GrainPattern` added. Allows adding grainniness effect to any element. [More info](https://www.amcharts.com/docs/v5/concepts/colors-gradients-and-patterns/patterns/#Grain_patterns).
- New private setting `trustBounds` added to `Sprite`. If set to `true`, the Sprite will check if a mouse pointer is within bounds of a Sprite before dispatching its pointer events. This helps to solve ghost tooltips problem which sometimes appears while moving the pointer over interactive objects. It is set to `true` by default on `Rectangle` and `Circle`.

### Changed
- Previously elements only supported one fill at a time - either `fill` or `fillGradient` or `fillPattern`. Now `fill` or `fillGradient` will be drawn even if `fillPattern` is set. This allows combining 
patterns with gradient fills.
- Gradients no longer inherit parent object's `fill`.
- It is now possible to draw annotations on a `StockChart` outside X-axis' min/max using axis overzoom.
- If a `forceHidden` for the tooltip is set to `true` the tooltip size won't be taken into account when arranging tooltips on an `XYChart`.

### Fixed
- An error was being thrown when chart with focused element was being disposed.
- If a series had data items without value and axis was zoomed so that only these data items were in range, the series used its full min and max values resulting value axis to be fully zoomed out.
- Using `hide()`/`show()` methods on an axis data item would no hide its visual elements.
- Drawing annotations on a `StockChart` with timezone set was a bit wonky.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
